src/Pirouette/Symbolic/Prover.hs

```diff
@@ -81,15 +81,15 @@ proveUnbounded opts = prove (opts {shouldStop = const False})
 --  means that on all paths that we looked at they were either verified or
 --  they reached the stopping condition.
 proveAny ::
-  (SymEvalConstr lang, PirouetteDepOrder lang m) =>
+  (SymEvalConstr lang, PirouetteDepOrder lang m, MonadIO m) =>
   Options ->
   (Path lang (EvaluationWitness lang) -> Bool) ->
   Problem lang ->
   m (Maybe (Path lang (EvaluationWitness lang)))
```

Comment:
Here I'm not sure what the monad is in practice. Adding the `MonadIO m` is enough to get things to compile so I only did that.